src/wp-includes/class-wp-block-patterns-registry.php

```diff
@@ -262,29 +262,3 @@ public static function get_instance() {
 		return self::$instance;
 	}
 }
-
```

Comment:
As above.

Reply:
The `register_block_pattern()` and `unregister_block_pattern()` functions were moved to the [`wp-includes/block-patterns.php`](https://github.com/WordPress/wordpress-develop/pull/3470/files#diff-b4514aa0e877e5200774c5cd82ec0870446f0a93ec7699eaa43c1a33c0d4f74a) file. That file, is required in `wp-settings.php`, so it's loaded anyway.
Do you think we'd still need to `require_once` it? :thinking: